src/asm_syntax.hpp

```diff
@@ -164,10 +176,10 @@ enum { T_UNINIT = -6, T_MAP = -5, T_NUM = -4, T_CTX = -3, T_STACK = -2, T_PACKET
 enum class TypeGroup {
     num,
     map_fd,
-    ctx,
-    packet,
-    stack,
-    shared,
+    ctx,            ///< pointer to the context
+    packet,         ///< pointer to the packet
+    stack,          ///< pointer to the stack
+    shared,         ///< pointer to shared memory
```

Comment:
These comments add very little IMO. If it is not clear that these are pointers, we should rename them to `ptr_ctx`, `ptr_packet` etc.

Also, "context" is vague  (for example the term might refer to pointer to a context in the verifier code itself) whereas "ctx" is actually more precise - special memory region named `ctx`.

Reply:
Updated ctx comment.  I agree renames would be helpful in general.